spice/urls.py

```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.conf.urls import include
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/', include('spice.account.urls')),
```

Comment:
The path should `path('api/acount', include('spice.account.urls')),`

Reply:
Fixed